.gitlab-ci.yml

```diff
@@ -0,0 +1,83 @@
+# This file is a template, and might need editing before it works on your project.
+# Official docker image.
+image: docker:git
+
+services:
+  - docker:dind
+
+stages:
+  - build
+  - test
+  - publish
+  - integration
+
+before_script:
+  - docker info
+  - git clone git://github.com/mendersoftware/mendertesting
+  - . mendertesting/check_commits.sh
```

Comment:
We should execute the script rather than source it. Sourcing it might have strange side effects

Reply:
I meant: `./mendertesting/check_commits.sh`

Reply:
thanks for catching that - I guess I meant to write `./mendertesting/check_commits.sh` - it's now `mendertesting/check_commits.sh` as you proposed + moved to a separate step